ts/features/itwallet/trustmark/machine/actions.ts

```diff
@@ -33,5 +56,38 @@ export const createItwTrustmarkActionsImplementation = (
     navigation.pop();
   };
 
-  return { onInit, handleSessionExpired };
+  /**
+   * Shows a failure toast
+   */
+  const showRetryFailureToast = ({
+    context
+  }: ActionArgs<Context, TrustmarkEvents, TrustmarkEvents>) => {
+    const timeDiffInSeconds = differenceInSeconds(
+      context.nextAttemptAt || new Date(),
+      new Date()
+    );
+
+    const time =
+      timeDiffInSeconds > 60
+        ? Math.ceil(timeDiffInSeconds / 60)
+        : timeDiffInSeconds;
+    const timeUnit =
+      timeDiffInSeconds > 60
+        ? I18n.t(`date.time.minute${time > 1 ? "s" : ""}`)
```

Comment:
Just a suggestion: maybe we can use [i18n-js pluralization](https://github.com/fnando/i18n?tab=readme-ov-file#pluralization)? Should be something like:
```js
I18n.t("date.time.minutes", { count: time })
```

The `it/index.yml`:
```yaml
minutes:
  one: ""
  other: ""
```

Reply:
I know, but it seems it does not work :/ 
Did you tried?

Reply:
Ok I think I resolved the issue, it works when you provide also a default value. See here: https://github.com/pagopa/io-app/pull/6558/commits/08c132838f16feb85db3e219ed7977985e4a44ed

Reply:
That `defaultValue` seems a bit redundant, but I see it's used like this in other sections of the app.